### ISSUE TITLE
Question: Is it possible to display results as json?

### ISSUE THREAD

**Issue by benrowe** [andjosh/clubhouse-cli#67]
How can i output the result of a command as json, or another more easily parseable format?

**Comment by j-martin** [comment]
Unless, I am mistaken no. It would not be hard to add though.

**Comment by andjosh** [comment]
Definitely possible!
I think we could do it with a formatting option like `%j` or `%json` for story output.

e.g.
```sh
$ club story 1234 --format %j
# outputs
{ entity_type: 'story',
  archived: false,
  created_at: '2019-03-01T02:54:27Z',
  updated_at: '2019-03-18T21:10:05Z',
  id: 4617,
  external_id: null,
  name:
   'Add profitability Slack alert for losing money',
  story_type: 'feature',
  position: 92161377280,
  workflow_state_id: 500000002,
  moved_at: '2019-03-18T21:10:05Z',
  started: true,
  started_at: '2019-03-18T16:16:36Z',
  started_at_override: null,
# ....
```

Would that work?

**Comment by benrowe** [comment]
That would be awesome. It would certainly work for me

My use case for this would be something like

`club story 1234 --format %j | jq '.comments.[-1].comment'`

Which would get the last comment on the story. This would make it trivial to post-process the result.

https://stedolan.github.io/jq/ for reference

**Comment by andjosh** [comment]
Released in [v1.17.0](https://github.com/andjosh/clubhouse-cli/blob/master/CHANGELOG.md#1170---2019-03-19)
